### PR TITLE
Improved content-disposition.js

### PR DESCRIPTION
**Pull request by def00111**

### CODE REVIEW

File: extension/content-disposition.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ function getFilenameFromContentDispositionHeader(contentDisposition) {
                     needsEncodingFixup = false;
                 }
             } catch (e) {
+                if (e instanceof TypeError && encoding == 'utf-8') {
+                    // Try iso-8859-1 encoding.
+                    return textdecode('iso-8859-1', value);
```
Comment (Rob--W) [inline extension/content-disposition.js]: Why this fallback?

**Comment by def00111** [comment]
> Is this an attempt to address #35 ?

Yes!

**Comment by Rob--W** [comment]
I plan to publish a new version relatively soon. When do you plan to respond to my review comments?

**Comment by Rob--W** [comment]
@def00111 Friendly reminder - see my previous comments.

**Comment by def00111** [comment]
> I plan to publish a new version relatively soon. When do you plan to respond to my review comments?

I do not use your add-on, but i like your content-disposition parser (i need it in my add-on).

**Pull request merged**